app.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+
+const { 
+    getTopics, 
+    getArticles
+} = require('./controllers/controllers.js');
+
+const app = express();
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.get(/^\/api\/articles/, getArticles);
```

Comment:
No need to use regex here as express already does this if we just pass a string of the endpoint, much like you have done for "/api/topics" above